tekton/src/tasks/fvt-launcher/launchfvt-mobile.yml.j2

```diff
@@ -87,6 +87,13 @@ spec:
               name: mas-fvt-mobile
```

Comment:
We have a Civil pipeline I created some time ago which is not used today... can we remove it as part of this PR of CLI, as now mobile will contain Civil things? (pipeline, launcher task, launcher playbook, etc?)

Reply:
Civil pipeline will coexist for some runs but will be removed later